File: Quizz_Objective-C/Quizz_Objective-C/ViewController.m
//
//  ViewController.m
//  Quizz_Objective-C
//
//  Created by m2sar on 31/01/2017.
//  Copyright © 2017 UPMC. All rights reserved.
//

#import "ViewController.h"
#import "MainView.h"

@interface ViewController ()

@end

@implementation ViewController


- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    _vue1 = [[MainView alloc] initWithFrame:[[UIScreen mainScreen]bounds]];
    _vue1.myController = self;
    self.view = _vue1;
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
  
}

- (void) viewWillTransitionToSize:(CGSize)size withTransitionCoordinator:(id<UIViewControllerTransitionCoordinator>)coordinator{
    [super viewWillTransitionToSize:size withTransitionCoordinator:coordinator];
    MainView *mv = (MainView*)self.view;
    [mv setPosition:size];
}

-(BOOL)shouldAutorotateToInterfaceOrientation:(UIInterfaceOrientation)interfaceOrientation
{
    return UIInterfaceOrientationIsLandscape(interfaceOrientation);
}


-(BOOL)shouldAutorotate{
    return YES;
}


@end

File: iSouvenir/David_Toty_TP7_iSouvenir/ViewController.swift
//
//  ViewController.swift
//  David_Toty_TP7_iSouvenir
//
//  Created by m2sar on 24/11/2016.
//  Copyright © 2016 UPMC. All rights reserved.
//

import UIKit
import CoreLocation

let v = MaVue(frame: UIScreen.main.bounds)

class ViewController: UIViewController {

    	
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        if CLLocationManager.locationServicesEnabled() {
            v.myController = self
            self.view = v
        } else  {
            self.view = UIView()
            let alert = UIAlertController(title: "Erreur", message: "Localisation désactivée", preferredStyle: .actionSheet)
            alert.addAction(UIAlertAction(title: "OK", style: .default, handler: nil))
            self.present(alert, animated: true, completion: nil)
        }
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

